home-service/src/main/java/com/softserveinc/ita/homeproject/homeservice/dto/InvitationDto.java

```diff
@@ -0,0 +1,16 @@
+package com.softserveinc.ita.homeproject.homeservice.dto;
+
+import java.time.LocalDateTime;
+import com.softserveinc.ita.homeproject.homedata.entity.Role;
+
+import lombok.Data;
+
+@Data
```

Comment:
Not sure about this annotation. It contains @Getter, @Setter, @RequiredArgsConstructor, @ToString and @EqualsAndHashCode but in other DTO we have @Getter, @Setter, @NoArgsConstructor and @AllArgsConstructor. Is it a good way to replace them with @Data? Probably we need to discuss it with @kh0ma.

Reply:
I think it's a good way. But if you want we can discuss it.

Reply:
Decided  to do it as it is now.